packages/cf-component-checkbox/test/Checkbox.js

```diff
@@ -1,8 +1,44 @@
 const React = require('react');
 const assertEqualJSX = require('assert-equal-jsx');
 const Checkbox = require('../src/Checkbox');
+const {expect} = require('chai');
+const {render, unmountComponentAtNode} = require('react-dom');
+const TestUtils = require('react-addons-test-utils');
+
+var then = function (callback, timeout) {
+  setTimeout(callback, timeout > 0 ? timeout : 0);
+  return {then: then};
+};
```

Comment:
So it turns out simulating events with `React TestUtils.simulate` [isn't synchronous](http://brandonokert.com/2015/08/04/TestingInReact/#You_Cannot_Check_State/Render_or_Simulate_Right_After_Simulating_an_Event).  The guy who wrote that article also provides some [utility patterns](http://brandonokert.com/2015/08/04/TestingInReact/#Utility_Patterns) which might be useful for us to extract into a lib so that we're consistent with how we test this stuff. 

In `<Notifications/>` I've seen other home grown methods for dealing with async testing scenarios.  I'm sure they exist else where.